packages/manager/apps/hycu/src/hooks/guide/useGuideUtils.tsx

```diff
@@ -2,7 +2,7 @@ import { useContext, useEffect, useState } from 'react';
 import { CountryCode } from '@ovh-ux/manager-config';
 import { ShellContext } from '@ovh-ux/manager-react-shell-client';
 
-type TGuideLinks = { [key in CountryCode]: string };
+type TGuideLinks = { [key in CountryCode | string]: string };
```

Comment:
I think it's a wrong type. This allow create { PAYS_UNKNOWN: "http://" }

Reply:
This is a workaround because MA doesn't exist in CountryCode so the project has build errors

Reply:
A better fix should be to add `MA` in the country code enum